src/EnergyPlus/ZoneContaminantPredictorCorrector.cc

```diff
@@ -1817,13 +1817,13 @@ void InitZoneContSetPoints(EnergyPlusData &state)
 
         // From decay model
         for (auto &con : state.dataContaminantBalance->ZoneContamGenericDecay) {
-            int Sch = ScheduleManager::GetCurrentScheduleValue(state, con.GCEmiRateSchedPtr);
-            if (Sch == 0.0 || state.dataGlobal->BeginEnvrnFlag || state.dataGlobal->WarmupFlag) {
+            int intSch = ScheduleManager::GetCurrentScheduleValue(state, con.GCEmiRateSchedPtr);
+            if (intSch == 0.0 || state.dataGlobal->BeginEnvrnFlag || state.dataGlobal->WarmupFlag) {
```

Comment:
So here, Sch shadows an outer REAL variable. So what was the intent? I *think* it was to convert a Real64 0 - 1 to either 0 or 1 given the text in the idd. Instead of changing this to read the schedule into a real I chose to int that data just like it was done before. It does seem like this was the intent, except the the idd says if it's 1 it will be set to 0 and here if it's 0 it's set to 0.

[src/EnergyPlus/ZoneContaminantPredictorCorrector.cc:1820]:(style),[shadowVariable],Local variable 'Sch' shadows outer variable

    ZoneContaminantSourceAndSink:Generic:DecaySource,
      A3 , \field Schedule Name
           \required-field
           \type object-list
           \object-list ScheduleNames
           \note Value in this schedule should be a fraction (generally 0.0 - 1.0) applied to the
           \note Initial Emission Rate. When the value is equal to 1.0, the time will be reset to
           \note zero.